tree/dataframe/inc/ROOT/RTrivialDS.hxx

```diff
@@ -34,6 +34,8 @@ protected:
 
 public:
    RTrivialDS(ULong64_t size, bool skipEvenEntries = false);
+   // This ctor produces a data-source that returns infinite entries
```

Comment:
```suggestion
   /// This ctor produces a data-source that returns infinite entries
```

Reply:
Is doxygen documentation desired for this class?

Reply:
Not really, RTrivialDS is just there for testing purposes. Honestly I don't even know why it's in the public namespace. Buuuut an extra slash doesn't hurt!